src/components/Home.module.css

```diff
@@ -35,16 +31,15 @@ body {
 
 .method{
   flex: 50%;
-  text-align: center-left;
   list-style-type: none;
   font-size: 1.5rem;
-  padding-left: 0%;
+  padding-left: 0px;
 }
 
 .aside {
   grid-area: aside;
   background-size: cover;
```

Comment:
I haven't checked but I'm guessing we apply an inline `background-image` style for this to be relevant?